### ISSUE TITLE
Suggestion: remove dash_pivottable.dev.js from package

### ISSUE THREAD

**Issue by rpkyle** [plotly/dash-pivottable#28]
The `dash_pivottable.dev.js` file appears currently unused by either the Python or R packages, and is approximately 20 MB in size. 

If possible, I'd suggest only building this additional JS via webpack when needed. For example, in `dash-renderer`'s `package.json` we have

```json
    "build:js": "webpack --build release",
    "build:dev": "webpack --build local",
```

which we use to control whether source maps are generated as well.

This recently caused friction with submitting the package to CRAN:

> _Thanks, we see:_
>
> _Size of tarball: 5900811 bytes_
>
> _Not more than 5 MB for a CRAN package, please._
>
> _Please fix and resubmit._

If we're able to make this change, the uncompressed R package size will plummet to 3 MB from over 20 MB, and the compressed archive will be approximately 1 MB.

@xhlulu @Marc-Andre-Rivet

**Comment by Marc-Andre-Rivet** [comment]
The dev build here just seems to be there to include source-maps. Seems ok to 🔪

**Comment by xhlulu** [comment]
Good idea for Manifest.in. I don't know how removing it from webpack.config.js would affect the build process, but if you both think it's a good idea let's do it.
